### PR TITLE
Expose thumbs up/down buttons in the message table

### PR DESCRIPTION
**Pull request by mehdi-zibout**
Initialized the values of thumbs up/down buttons (the "+1 and "-1" keys in the emojiState objects) to 0. also sorted `Object.entries(emojiState.emojis)` so the location of the buttons is consistent.

### CODE REVIEW

File: website/src/components/Messages/MessageTableEntry.tsx [hunk 1]
```diff
@@ -102,14 +109,18 @@ export function MessageTableEntry({ message, enabled, highlight }: MessageTableE
...
-              <MessageEmojiButton
-                key={emoji}
-                emoji={{ name: emoji, count }}
-                checked={emojiState.user_emojis.includes(emoji)}
-                onClick={() => react(emoji, !emojiState.user_emojis.includes(emoji))}
-              />
-            ))}
+            .sort(([emoji]) => -emoji)
+            .map(([emoji, count]) => {
+              console.log(emoji);
```
Comment (AbdBarho) [inline website/src/components/Messages/MessageTableEntry.tsx:114]: could you please remove the console log

Comment (mehdi-zibout) [reply website/src/components/Messages/MessageTableEntry.tsx:114]: Sorry I forgot it 😅, while I'm at it should I show the count only when it's greater than 0?

Comment (AbdBarho) [reply website/src/components/Messages/MessageTableEntry.tsx:114]: kind of, yeah, but not for the `+1` or `-1`, otherwise we are back to square one

Comment (mehdi-zibout) [reply website/src/components/Messages/MessageTableEntry.tsx]: image
done

**Review by AbdBarho — APPROVED** [review]

**Pull request merged**